sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/csv/CSVInferSchemaSuite.scala

```diff
@@ -267,7 +267,9 @@ class CSVInferSchemaSuite extends SparkFunSuite with SQLHelper {
   test("SPARK-45433: inferring the schema when timestamps do not match specified timestampFormat" +
     " with only one row") {
     val options = new CSVOptions(
-      Map("timestampFormat" -> "yyyy-MM-dd'T'HH:mm:ss"),
+      Map(
+        "timestampFormat" -> "yyyy-MM-dd'T'HH:mm:ss",
+        "timestampNTZFormat" -> "yyyy-MM-dd HH:mm:ss"),
```

Comment:
To infer the STRING type, the input must not match to **both** formats: `TIMESTAMP` and `TIMESTAMP_NTZ`. Just set `timestampFormat` is not enough.